hooks/useKeyboard/useKeyboard.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+// eslint-disable-next-line simple-import-sort/imports
+import { useEffect, useState, KeyboardEvent, KeyboardEventHandler } from 'react';
```

Comment:
the same as https://github.com/Juratbek/udas_frontend/pull/596#pullrequestreview-1244951022